telegram/ext/filters.py

```diff
@@ -74,15 +78,26 @@ def __repr__(self):
         return self.name
 
     def filter(self, message):
+        """
+        This method must be overwritten.
+
+        Args:
+            message (:class:`telegram.Message`): The message that is tested.
+
+        Returns:
+            :obj:`bool`
+        """
+
         raise NotImplementedError
 
 
 class InvertedFilter(BaseFilter):
-    """Represents a filter that has been inverted.
-
-    Args:
-        f: The filter to invert
-    """
+    # """Represents a filter that has been inverted.
```

Comment:
I uncommented them. Now I'm using autoclass, and not added these classes (mergedfilter and invertedfilter) to any toctree it's pretty good to have them just there.